source/deployment-guide/mobile/configure-microsoft-intune-mam.rst

```diff
@@ -0,0 +1,398 @@
+
+Getting Started
+---------------
+
+This configuration spans identity, mobile enforcement, and licensing systems. The guide is intentionally explicit to prevent misconfiguration and destructive enrollment failures. It's organized to help you validate fit first, then configure Intune MAM correctly.
+
+* Initial sections help you determine whether Intune MAM is compatible with your deployment.
+* `Identity <#identity-configuration-for-intune-mam>`__ sections explain the required identity model and enforcement behavior.
+* Configuration sections provide a prescriptive order of operations.
+* `Validation <#validation-checklist>`__ and `Troubleshooting <#troubleshooting>`__ describe expected runtime behavior and failure modes.
+
+When Not to Use This Guide
+---------------------------
+
+If any of the following apply, stop. This configuration will fail.
+
+* You require Android Intune MAM support (not yet available).
+* Your deployment cannot use Microsoft Entra ID (Azure AD).
+* Your identity strategy cannot use Azure AD ``objectId`` as the authoritative user identifier.
+* You need a rollout model where users can defer or bypass Intune enrollment.
+
+Before You Continue
+-------------------
+
+Before proceeding, confirm the following are true:
+
+* You use Microsoft Entra ID for authentication.
+* You can commit to Azure AD ``objectId`` as the authoritative identity.
+* You have (or can obtain) a Mattermost Enterprise Advanced license.
+* Target users are licensed for Microsoft Intune.
+* You can register applications and grant admin consent in Microsoft Entra.
+
+If any of the above are not true, do not proceed.
+
+Configuration Overview
+----------------------
+
+Configuring Intune MAM for Mattermost Mobile requires coordinated setup across the following four systems:
+
+* **Microsoft Entra ID (Azure AD)** – identity, app registration, API permissions
+* **Microsoft Intune** – app protection policies and user targeting
+* **Mattermost Server** – MAM enablement and identity alignment
+* **Mattermost Mobile App (iOS)** – enrollment and enforcement
+
+If any system is misconfigured, Intune MAM enrollment will fail.
+
+Before beginning configuration, review the `Identity Configuration <#identity-configuration-for-intune-mam>`__ section to confirm your deployment meets the required identity model.
+
+Setup Order
+~~~~~~~~~~~
+
+Follow this setup order exactly to avoid enrollment failures and rework.
+
+Step 1: Finalize Identity Strategy (Entra + Mattermost)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* Commit to Azure AD ``objectId`` as the authoritative identity.
+* Ensure all authentication methods (OAuth, SAML, LDAP) resolve to the same value.
```

Comment:
```suggestion
* Ensure all authentication methods (OpenID Connect or SAML, LDAP) resolve to the same value.
```

OpenID Connect will always be `objectId` while SAML and LDAP need configured to use the value of `objectId`, at the same time SAML is only required to be that way if it is the selected `Auth Provider` in the Intune MAM configuration in the admin console.